inc/class-upgrade.php

```diff
@@ -853,6 +854,22 @@ private function upgrade_169() {
 		}
 	}
 
+	/**
+	 * Performs the 17.0 upgrade. shop_order indexables were cleaned from the indexable table in 16.9, so we have to clean out the orphaned entries from the rest of the tables.
+	 *
+	 * @return void
+	 */
+	private function upgrade_170() {
+		// Sets a scheduled job to do the cleanup eventually and not in the upgrade process itself. When that cleanup is completed, the job will de-register itself.
+		if ( ! wp_next_scheduled( 'wpseo_cleanup_orphaned_indexables' ) ) {
+			wp_schedule_event(
+				time(),
+				'hourly',
+				'wpseo_cleanup_orphaned_indexables'
```

Comment:
make sure to name this whatever we end up naming it

Reply:
I think the crons themselves should be relevant to the task at hand. If in the future we want to do another cleanup, we will register new crons anyway.

See also: https://github.com/Yoast/wordpress-seo/pull/17314#discussion_r683519805